easybuild/easyblocks/generic/binariestarball.py

```diff
@@ -40,21 +41,43 @@ class BinariesTarball(Tarball):
     """
     Support for installing a tarball of binaries
     """
+    @staticmethod
+    def extra_options(extra_vars=None):
+        """
+        Define list of files to be copied during the installation process
+        """
+        extra_vars = Tarball.extra_options(extra_vars)
+        # Allow specification of files to copy explicitly.
+        # If files_to_copy is a zero-length list, all files (but not
+        # directories, etc.) in start_dir will be copied with their
+        # current names.
+        extra_vars['files_to_copy'] = [
+            [],
+            "List of optional (source_file, destination_file) tuples",
+            CUSTOM,
+        ]
+        return extra_vars
 
     def install_step(self):
         """Install by copying unzipped binaries to 'bin' subdir of installation dir, and fixing permissions."""
 
         bindir = os.path.join(self.installdir, 'bin')
+        items_to_copy = []
         try:
             os.makedirs(bindir)
-            for item in os.listdir(self.cfg['start_dir']):
+            if len(self.cfg['files_to_copy']) == 0:
```

Comment:
Only if initialising the value to None (see above) is considered more desirable.